openlibrary/core/lending.py

```diff
@@ -50,7 +50,10 @@ def setup(config):
     global config_content_server, config_loanstatus_url, config_ia_access_secret, config_bookreader_host, config_ia_ol_shared_key
 
     if config.get("content_server"):
-        config_content_server = ContentServer(config.get("content_server"))
+        try:
+            config_content_server = ContentServer(config.get("content_server"))
+        except:
```

Comment:
What kind of exception are we expecting? A KeyError or ValueError?
Also, is there any way for us to log when this happens?
What happens if the config_content_server is then later accessed elsewhere (and it is not set)? Is there a try/except around that? Should config_content_server have a default value?